### PR TITLE
Added Superchain time in Binary mainnet 

### PR DESCRIPTION
**Pull request by priyanshuthapliyal55**
# Adding a New Chain
This PR adds [Chain Name Here] to the registry.

## Checklist
- [x] I have declared the chain at the appropriate [Superchain Level](../docs/glossary.md#superchain-level-and-rollup-stage).
- [x] I have run `just validate <chain-id>` locally to ensure all local validation checks pass. 
- [x] I have run `just codegen` to ensure that the chainlist and other generated files are up-to-date and include my chain.

### CODE REVIEW

**Review by sebastianst — CHANGES_REQUESTED** [review]
Please rebase this PR so it's a single small commit on top of `main`.

**Comment by priyanshuthapliyal55** [comment]
> Please rebase this PR so it's a single small commit on top of `main`.

File: superchain/configs/mainnet/tbn.toml [hunk 1]
```diff
@@ -10,6 +10,7 @@ delta_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
 ecotone_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
 fjord_time = 1725536344  #Thu 05 Sep 2024 11:39:04 UTC
 granite_time = 1726070401 # Wed 11 Sep 2024 16:00:01 UTC
+superchain_time = 1736445601 # Holocene - ​Thu Jan 9 at 18:00:01 UTC
```
Comment (geoknee) [inline superchain/configs/mainnet/tbn.toml:13]: Could you add the year to the comment?

Comment (geoknee) [reply superchain/configs/mainnet/tbn.toml:13]: We shouldn't associate this time with Holocene per se. What this declaration means is that form this time onwards, _all_ following hardforks will be inherited automatically. Only when the mainnet superchain Holocene time has been committed https://github.com/ethereum-optimism/superchain-registry/pull/726 would Binary mainnet would receive that time.

Comment (priyanshuthapliyal55) [reply superchain/configs/mainnet/tbn.toml:13]: so what timestamp should i use so that Binary mainnet will activate holocene as well as future hardfork?

Comment (geoknee) [reply superchain/configs/mainnet/tbn.toml]: This should be fine, we are just not yet seeing the net effect on downstream node software until the superchain Holocene time is fully locked in.

You could tweak the comment to say "from Holocene onwards" to reduce confusion.

File: superchain/configs/mainnet/tbn.toml [hunk 2]
```diff
@@ -10,6 +10,7 @@ delta_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
 ecotone_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
 fjord_time = 1725536344  #Thu 05 Sep 2024 11:39:04 UTC
 granite_time = 1726070401 # Wed 11 Sep 2024 16:00:01 UTC
+superchain_time = 1736445601 # Holocene - ​Thu Jan 9 2024 18:00:01 UTC
```
Comment (geoknee) [inline superchain/configs/mainnet/tbn.toml:13]: Do you mean 2025?

Comment (priyanshuthapliyal55) [reply superchain/configs/mainnet/tbn.toml]: oh sorry i have updated it

**Comment by sebastianst** [comment]
Will let tbn join mainnet superchain times from Granite in https://github.com/ethereum-optimism/superchain-registry/pull/726, so closing this PR

**Pull request closed without merging**